### ISSUE TITLE
Change note editing background to solarized, dark or light

### ISSUE THREAD

**Issue by Hedronmx** [streetwriters/notesnook#252]
**Is your feature request related to a problem? Please describe.**
Not at all

**Describe the solution you'd like**
It would be nice to have the option to set the color of the editing windows background. For example, I really like the light theme but typing on a light background is hard. So i could choose to change the editing background to a solarized theme or to a dark theme. 

That way I still have the light theme for the rest of the app but when typing i have other options.

**Describe alternatives you've considered**
Switching between light and dark mode across the whole app

**Additional context**
Something like this, where the whole app is light themed but the editing window is other color

![Screenshot from 2021-12-31 12-43-39](https://user-images.githubusercontent.com/26935602/147838536-e996b5fc-6adc-4b03-8e86-1c15c0c8219a.png)
![Screenshot from 2021-12-31 12-44-25](https://user-images.githubusercontent.com/26935602/147838554-42fe6425-f829-4184-a69a-cdd272301893.png)

**Comment by thecodrr** [comment]
This should be completed after #2196 is merged.
